src/app/Library/CrudPanel/ObjectGroup.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel;
+
+class ObjectGroup
```

Comment:
Let's use `CrudObjectGroup` please, like we talked on Skype. It's too long but nobody has to remember it because everybody's using:

```php
$this->crud->group()
// or
CRUD::group()
```

Reply:
Done \o